Formula/m/mcp-server-kubernetes.rb

```diff
@@ -0,0 +1,29 @@
+class McpServerKubernetes < Formula
+  desc "MCP Server for kubernetes management commands"
+  homepage "https://github.com/Flux159/mcp-server-kubernetes"
+  url "https://registry.npmjs.org/mcp-server-kubernetes/-/mcp-server-kubernetes-2.9.1.tgz"
+  sha256 "83dc88067745fe32791706babdaab7fd84d6a0a6a582dcb3a2904a20d8c35821"
+  license "MIT"
+
+  depends_on "node"
+
+  on_linux do
+    depends_on "glibc" => :build
+  end
```

Comment:
Hum, i don't know how to fix this: https://github.com/Homebrew/homebrew-core/actions/runs/17712042176/job/50332084893?pr=241641#step:4:76

It was not happening for me since i used it only on macOS :-/